### ISSUE TITLE
Unable to close the "TRACING" and "QUERY PLAN" panes

### ISSUE THREAD

**Issue by DarkLite1** [prisma-labs/graphql-playground#1264]
#### This issue pertains to the following package(s):

- [x] GraphQL Playground HTML
- [x] apollo-server-express: "^2.16.0",

#### What OS and OS version are you experiencing the issue(s) on?

Windows Server 2016

#### What version of graphql-playground(-electron/-middleware) are you experiencing the issue(s) on?

We're using `apollo-server-express` version `^2.16.0` which comes with the Playground build in.

#### What is the actual behavior?

When exploring the GUI I tried to click on the words TRACING and QUERY PLAN. But now I am unable to close it again.

![image](https://user-images.githubusercontent.com/10828570/90748440-7ad4d700-e2d2-11ea-8078-5c62341b02a5.png)

#### What steps may we take to reproduce the behavior?

Run a standard ApolloServer.

**Comment by Bilal-Z** [comment]
I had the same issue, but if you hover your mouse right above where it says Tracing and Query Plan you will notice the cursor changes to an up or down arrow depending upon if the pane is open or closed. see attached video.

https://user-images.githubusercontent.com/10467233/103271738-df204980-49dc-11eb-98b1-ad702dc46e40.mp4

**Comment by DarkLite1** [comment]
You are correct, when you hover over the border you can see the small arrow to collapse the panes. It would be better however to have s mall cross like on a file explorer close button. So leaving this open for improvement for now.
